### ISSUE TITLE
[Bug] There is no label option under series->custom  confuguration

### ISSUE THREAD

**Issue by BrazilAmando** [apache/echarts#19648]
### Version

5.5.0

### Link to Minimal Reproduction

https://echarts.apache.org/en/option.html#series-custom.type

### Steps to Reproduce

visit chart configuration and you cannot find label option although label option works.
![image](https://github.com/apache/echarts/assets/125205627/baa475e4-3ab0-4f6d-a962-10760a65f623)

### Current Behavior

series->custom->label option works

### Expected Behavior

add label configuration to the chart configuration

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by helgasoft** [comment]
yes, _label_ is missing in the API docs, but works - [Demo Code](https://echarts.apache.org/examples/en/editor.html?code=PYBwLglsB2AEC8sDeAoWsAmBDMWDOApmAFywDaa6ylVmEAtgdHlM6WQOQAeHANLBwCeHALq8aVPMACuAJwDGBdhNpkAtAEZ-AZjEqqZLbACse2qqOaR-s1QC-49La4BBLhDykkDyoLcevH3RCWQgCT3IJVHNYMEEQJQF5aTwwYHo-fWgsRlIOZNT0zJjZJgwCWQBJMAJ6AApS6HKq6AAzYH4sEAgASmoY9HkYKQAbAgA6EeAAcwayisq24HGmIfKegG59QeGwWAA3DQRYLohx_awR6QI5poWlleg1ia4yAAYRTe3YIeY9_YATMdTudLtdbs1Fu1Hs9xoJ3p8tgMfrtyFx-IIRMDuuMhsBZBg6mRDvxAYjvqUwHI4NFkbF4ol8hAFGNinT0d90IJHHS8AALLqJWl0n5cUhvHki-SCcWSumyUjGTmwIK8uJjUgg1KCMZ1HrfOxI8yq2hDKYKgQAIyuBD4sH0IywloIIy8_OAAHdSGBZNd-MBWq1CCRDAAWXgfFUSVXWQ1AA)

**Comment by BrazilAmando** [comment]
Thanks for your reply. I am a fresher otherwise I would correct this bug.
